### ISSUE TITLE
[Feature Request] Register as an equalizer app to respond to related intents

### ISSUE THREAD

**Issue by Venderbad** [ThePBone/RootlessJamesDSP#95]
Some local music player (Retro Music Player for instance) offers a button to instantly switch to the external equalizer app. On LineageOS the button will get me straight to AudioFX, which is a system app in my LineageOS. I think it would be nice to have RootlessJamesDSP to respond to that intent since this app is much more powerful than AudioFX which I seldomly use.

This app is amazing by the way, it is a perfect alternative to Viper4Android since v4a is damn hard to install and depends on root privilege. The app offers similar functions without compromising almost anything crucial. Many thanks to the great devs!:grinning:

**Comment by ThePBone** [comment]
Hi! This app already does this and responds to these intents. However, I did some experimenting and found out that if a privileged system EQ application is installed on the phone, these intents will always be delivered to that one instead. It won't even show the app selection dialog in that case.

Only after I disabled the system EQ app on my phone using ADB, Android showed me a selection dialog of non-system EQ apps installed on my phone (including RootlessJamesDSP).

So if you wanted to make this work, the only way to fix this would be to disable/uninstall the AudioFX app because that one always takes precedence over user-level apps.
